packages/theme-chalk/src/table.scss

```
@@ -235,6 +235,11 @@
     &.gutter {
       width: 0;
     }
+
+    .cell {
```

```suggestion
    &.cell {
```

It may be better to keep a uniform format